addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -371,6 +371,23 @@ var MassMailingFieldHtml = FieldHtml.extend({
 
         this.wysiwyg.$iframeBody.find('.iframe-utils-zone').addClass('d-none');
 
+        // Templates taken from old mailings
+        const result = await this._rpc({
+            model: 'mailing.mailing',
+            method: 'action_fetch_favorites',
+        });
+        const templatesParams = result.map(values => {
+            return {
+                id: values.id,
+                name: `template_${values.id}`,
+                subject: values.subject,
+                template: values.body_arch,
+                userId: values.user_id[0],
+                userName: values.user_id[1],
+                nowrap: true,
```

Comment:
Linting: nowrap should be defined before subject :D other keys are well sorted alphabetically, let us do the last one

Reply:
yes exactly, those HTML element are save in the body_arch field and contains background color and stuff like that. As we also want to copy them, we do not need to encapsulate again the template

Reply:
OK thanks for the explanation :)